dbt_platform_helper/utils/application.py

```diff
@@ -104,6 +115,17 @@ def is_environment_key(name):
         ]
     }
 
+    response = ssm_client.get_parameters_by_path(
+        Path=f"/copilot/applications/{application.name}/components",
+        Recursive=False,
+        WithDecryption=False,
+    )
+
+    application.services = {
+        svc["name"]: Service(svc["name"], svc["type"])
+        for svc in [json.loads(p["Value"]) for p in response["Parameters"]]
```

Comment:
Personal preference, but I would find that the following requires slightly less thinking...
```suggestion
        for svc in [json.loads(parameter["Value"]) for parameter in response["Parameters"]]
```